posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,600 @@
+
+
+The full list of specifications that make up the three profiles is as follows:
+
+[#coreprofile]
+== Jakarta EE Core Profile 10
+[cols="4,2,3",options="header"]
+|===
+|Specification |Updates |Feature Documentation
+
+[[jsonb]]
+|https://github.com/eclipse/microprofile-open-api/releases/tag/3.1[JSON Binding 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/jsonb-3.0.html[jsonb-3.0]
+
+[[rest]]
+|link:https://jakarta.ee/specifications/restful-ws/3.1/jakarta-restful-ws-spec-3.1.html[RESTful Web Services 3.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/restfulWS-3.1.html[restfulWS-3.1], link:{url-prefix}/docs/latest/reference/feature/restfulWS-3.1.html[restfulWSClient-3.1]
+
+[[jsonp]]
+|https://github.com/eclipse/microprofile-metrics/releases/tag/5.0[JSON Processing 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/jsonp-2.1.html[jsonp-2.1]
+
+[[cdi]]
+|https://jakarta.ee/specifications/annotations/1.3/[Annotations 1.3] & https://jakarta.ee/specifications/interceptors/2.1/[Interceptors 2.1]
+|Minor updates
+|link:{url-prefix}/docs/latest/reference/feature/cdi-4.0.html[cdi-4.0]
+
+[[cdi]]
+|https://jakarta.ee/specifications/annotations/1.3/[Dependency Injection 2.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/cdi-4.0.html[cdi-4.0]
+
+|===
+
+NOTE: Jakarta EE Core Profile 10 also introduces CDI Lite 4.0, which is a subset of the features provided by CDI 4.0 and is therefore made available as part of the link:{url-prefix}/docs/latest/reference/feature/cdi-4.0.html[cdi-4.0] feature.
+
+[#webprofile]
+== Jakarta EE Web Profile 10
+[cols="4,2,3"]
+|===
+|Specification |Updates |Liberty Feature Documentation
+
+|https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE 10 Core Profile]
+|New
+|<<coreprofile, See previous table>>
+
+|https://jakarta.ee/specifications/authentication/3.0/[Authentication 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/appAuthentication-3.0.html[appAuthentication-3.0]
+
+|https://jakarta.ee/specifications/cdi/4.0/[Context Dependency Injection 4.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/cdi-4.0.html[cdi-4.0]
+
+|https://jakarta.ee/specifications/concurrency/3.0/[Concurrency 3.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/concurrent-3.0.html[concurrent-3.0]
+
+|https://jakarta.ee/specifications/expression-language/5.0/[Expression Language 5.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/expressionLanguage-5.0.html[expressionLanguage-5.0]
+
+|https://jakarta.ee/specifications/faces/4.0/[Expression Language 5.0]

```

Comment:
```suggestion
|https://jakarta.ee/specifications/faces/4.0/[Jakarta Server Faces 4.0]

```

Reply:
If you are listing specs and linking to the docs then I think you should give them all their formal names, which start with "Jakarta".

Reply:
```suggestion
|https://jakarta.ee/specifications/faces/4.0/[Jakarta Faces 4.0]
```

`Server` was removed from the name.

Reply:
Well that'll teach me for using the spec as a reference!
https://jakarta.ee/specifications/webprofile/10/jakarta-webprofile-spec-10.0.html#a43

Reply:
https://jakarta.ee/specifications/faces/4.0/ has it right.  Looks like a small defect in the jakarta platform repo is in order to fix that.

Reply:
I did initially, but then it looked extremely repetitive since all of them started with "Jakarta" and the table title already includes "Jakarta".  But I'm not opposed to adding "Jakarta" back in if we feel we need it.

Reply:
I think we should ... you included "MicroProfile" on the front of all of those spec names